apps/studio/tsconfig.json

```diff
@@ -28,7 +28,8 @@
         "./src/*",
         "./public/*"
       ]
-    }
+    },
+    "types": ["cypress"]
```

Comment:
Required so that the Next.js type checker is happy, see https://docs.cypress.io/app/tooling/typescript-support#Configure-tsconfigjson.